docs/ref/unicode.txt

```diff
@@ -35,58 +36,36 @@ able to store certain characters in the database, and information will be lost.
 .. _section 2: https://docs.oracle.com/database/121/NLSPG/ch2charset.htm#NLSPG002
 .. _section 11: https://docs.oracle.com/database/121/NLSPG/ch11charsetmig.htm#NLSPG011
 
-All of Django's database backends automatically convert strings into
+All of Django's database backends automatically convert text into bytes using
```

Comment:
I'd like to understand this better. Could you point out an example of where it happens? I searched `django/db` for `encode(` but didn't find anything.